.github/workflows/dotnet.yml

```diff
@@ -37,5 +37,5 @@ jobs:
     - name: Build package
       run: dotnet pack -c Release -o ./output
     - name: Publish
-      run: dotnet nuget push ./output/Duffel.ApiClient.*.nupkg -k ${{secrets.NUGET_API_KEY}}
+      run: dotnet nuget push ./output/Duffel.ApiClient.*.nupkg -k ${{secrets.NUGET_API_KEY}} -s 'https://api.nuget.org/v3/index.json'
```

Comment:
got the source from https://docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-the-dotnet-cli#publish-with-dotnet-nuget-push